react/table/src/components/pagination/PaginationWrapper.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import { withTranslation } from 'react-i18next';
+import TablePagination from '@material-ui/core/TablePagination';
+
+import TablePaginationActions from 'components/pagination/TablePaginationActions';
+
+import conferenceType from 'components/__types__/conference';
+
+const PaginationWrapper = ({
+  children,
+  pagination,
+  paginationMode,
+  onLoadMore,
+  items,
+  changePage,
+  changeItemsPerPage,
+  t,
+}) => {
+  if (paginationMode === 'infinite-scroll') {
+    return (
+      <InfiniteScroll
+        dataLength={items.length}
+        next={onLoadMore}
+        hasMore={items.length < pagination.itemCount}
+        loader={<div>{t('common.loadingMore')}</div>}
+      >
+        {children}
+      </InfiniteScroll>
+    );
+  }
+  if (paginationMode === 'pagination') {
+    return (
+      <>
+        {children}
+        <TablePagination
+          component="div"
+          rowsPerPageOptions={[5, 10, 25, { label: 'All', value: -1 }]}
```

Comment:
these values are hardcoded, while rowPerPage in the reducer's initialState is parametric